keyboards/wheatfield/blocked65/blocked65.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+#include "config.h"
```

Comment:
```suggestion
```
`config.h` is included by default and so this line shouldn't be needed.

Reply:
Fixed